### DISCUSSION TITLE
why the IINA not updated

### DISCUSSION THREAD

**Discussion by rookievips** [iina/iina#3726] [Q&A] [answered]
### Why is IINA not updated,IINA hasn't had an official update in almost a year

**Comment by tpfortree** [comment]
and why there's so many issues : 1.3K ,,,  is the author give it up ?

**Reply by saagarjha** [reply]
I don't think this is particularly surprising considering IINA has hundreds of thousands of users.

**Comment by saagarjha** [comment, marked as answer]
IINA will be updated when there's enough work done to cut an update. In the meantime, I hope you enjoy using your current version, and please don't hesitate to file issues if you run into problems.

**Reply by MurtadhaInit** [reply]
Cheers guys, I think IINA is absolutely wonderful. Keep up the good work! 🙂

**Reply by tzm41** [reply]
Looking forward to the HDR features!

**Comment by Danann36664** [comment]
Looking forward to the update! It is a great and nice app for macOS user!

**Comment by low-batt** [comment]
IINA 1.3.0 has been released. It can be downloaded from the official [IINA](https://iina.io/) website.
